templates/data/skr-svcat-migration-data.yaml

```diff
@@ -0,0 +1,42 @@
+templates:
+  - fromTo:
+      - from: templates/generic.tmpl
+        to: ../prow/jobs/kyma/skr-svcat-migration.yaml
+    render:
+      - localSets:
+          jobConfig_default:
+            annotations:
+              pipeline.clusterprovisioning: keb
+              pipeline.installer: keb
+              pipeline.test: fast-integration
+            path_alias: github.com/kyma-project/kyma
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/skr-svcat-migration.bash"
```

Comment:
In favour of separate script for that you can define the script execution directly here.
```
command: "make"
args:
    - "-C"
    - "/home/prow/go/src/github.com/kyma-project/kyma/tests/fast-integration ci-skr-svcat-migration"
```

Reply:
thanks, applied in: https://github.com/kyma-project/test-infra/pull/3913/commits/b912001dafd9abf6e158852502adaf07763cc66b
tested through https://status.build.kyma-project.io/log?container=test&id=1418513673479000064&job=wozniakjan_test_of_prowjob_skr-azure-svcat-migration-dev